### ISSUE TITLE
Parsing errors thown on cppcheck variableScope warnings

### ISSUE THREAD

**Issue by qwhelan** [jbenden/vscode-c-cpp-flylint#90]
`cppcheck` sometimes outputs multi-line messages of the form:
```
  `"/home/chris/code/bottleneck/bottleneck/src/move_median/move_median.c  539  style variableScope: The scope of the variable 'idx2' can be reduced. Warning: Be careful when fixing this message, especially when there are inner loops. Here is an example where cppcheck will write that the scope for 'i' can be reduced:`,
  'void f(int x)',
  '{',
  '    int i = 0;',
  '    if (x) {',
  "        // it's safe to move 'int i = 0;' here",
  '        for (int n = 0; n < 10; ++n) {',
  "            // it is possible but not safe to move 'int i = 0;' here",
  '            do_something(&i);',
  '        }',
  '    }',
  '}',
  'When you see this message it is always safe to reduce the variable scope 1 level."',
```
Note that the first line has the opening `"` and the ending line has the closing `"`. I believe the first line is successfully parsed, but the next 12 are not, causing the parsing to abort due to too many failures.

A possible solution would be to have `getLines()` concatenate lines until an EOL closing quote is found.

**Comment by jbenden** [comment]
Hello,

Cppcheck is not suppose to output multiple line messages, unless the parameter `--template-location` is specified. Did this flag get passed somehow?

If it's still an issue, please enable the `c-cpp-flylint.debug` settings, then post the entire contents of the `Output` -> `C/C++ Flylint` messages window.

Thanks!
-Joe

**Comment by qwhelan** [comment]
Here's the output from one run - no `--template-location` in the call:
```
executing:  /usr/bin/cppcheck --inline-suppr --enable=warning,style,performance,portability,information -I /home/chris/code/bottleneck/bottleneck/src -I /home/chris/anaconda3/include/python3.7m/ -I /home/chris/anaconda3/lib/python3.7/site-packages/numpy/core/include -I /usr/include -I /usr/include/sys -I /usr/include/newlib -I /usr/include/clang/10/include --suppress=variableScope:/home/chris/anaconda3/lib/python3.7/site-packages/numpy/core/include/numpy/_neighborhood_iterator_imp.h --language=c --platform=native --template="{file}  {line}  {severity} {id}: {message}" --verbose --force --inconclusive /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h
[
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h ...',
  'Defines:',
  'Undefines:',
  'Includes: -I/home/chris/code/bottleneck/bottleneck/src/ -I/home/chris/anaconda3/include/python3.7m/ -I/home/chris/anaconda3/lib/python3.7/site-packages/numpy/core/include/ -I/usr/include/ -I/usr/include/sys/ -I/usr/include/newlib/ -I/usr/include/clang/10/include/',
  'Platform:Native',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: BINARY_TREE=1...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: MB_LEN_MAX...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: NAN...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: NDEBUG...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: NDEBUG;__USE_GNU...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: PATH_MAX;_LIBC_LIMITS_H_...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _ASSERT_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _ASSERT_H;__USE_GNU...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BITS_STDIO2_H;_STDIO_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BITS_STDIO_H;_STDIO_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BITS_STDIO_LIM_H;_STDIO_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BITS_STRING_FORTIFIED_H;_STRING_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BITS_TYPES_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BSD_SOURCE;_DEFAULT_SOURCE;_SVID_SOURCE...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _BYTESWAP_H;_ENDIAN_H;_NETINET_IN_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _ENDIAN_H;_NETINET_IN_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _FILE_OFFSET_BITS...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _FORTIFY_SOURCE;__OPTIMIZE__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _GETOPT_H;__USE_GNU;__USE_POSIX2;__USE_POSIX_IMPLICITLY...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _GETOPT_H;__USE_GNU;__USE_POSIX2;__USE_POSIX_IMPLICITLY;__REDIRECT...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _GNU_SOURCE...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _MATH_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _MSC_EXTENSIONS;_NATIVE_NULLPTR_SUPPORTED;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _MSC_EXTENSIONS;_WCHAR_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__has_feature=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _MSC_VER...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _MSC_VER;__MINGW32__;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _Noreturn;__GNUC_PREREQ=0;__STDC_VERSION__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _OPENMP;__FAST_MATH__;__x86_64__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _PTRDIFF_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _PTRDIFF_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__has_feature=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _REENTRANT;_THREAD_SAFE...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _RSIZE_T;__STDC_WANT_LIB_EXT1__;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _RSIZE_T;__STDC_WANT_LIB_EXT1__;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__has_feature=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _SIZE_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _SIZE_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__has_feature=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _STDIO_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _STDIO_H;_UNISTD_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _STDLIB_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _STDLIB_H;_SYS_WAIT_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _SYS_SELECT_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _SYS_WAIT_H...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _Static_assert;__GNUC_PREREQ=0;__STDC_VERSION__;__STRICT_ANSI__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _WCHAR_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _WCHAR_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__has_feature=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _WINT_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__has_feature=0;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: _WINT_T;__STDDEF_H;__need_NULL;__need_ptrdiff_t;__need_wchar_t;__need_wint_t;__need_wint_t...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __APPLE__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __ASSEMBLER__;__USE_MISC...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __CUDACC__;__GNU__;__ICC;__x86_64__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __FAST_MATH__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __FAST_MATH__;__x86_64__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __FLT_EVAL_METHOD__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __FP_FAST_FMA;__USE_ISOC99...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __FP_FAST_FMAF;__USE_ISOC99...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __FP_FAST_FMAL;__USE_ISOC99...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GCC_IEC_559...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GCC_IEC_559_COMPLEX...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GLIBC_INTERNAL_STARTING_HEADER_IMPLEMENTATION...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GLIBC_USE=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GLIBC_USE=0;__LDBL_COMPAT...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GLIBC_USE=0;__LDBL_COMPAT;__REDIRECT...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GLIBC_USE=0;__REDIRECT...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GLIBC_USE=0;__USE_ISOC99...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_GNU_INLINE__;__GNUC_STDC_INLINE__;__clang__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_GNU_INLINE__;__GNUC_STDC_INLINE__;__clang__;__GNUC_STDC_INLINE__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ;__GNUC_PREREQ;__GNUC_PREREQ...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ=0...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ=0;__HAVE_FLOAT64X...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ=0;__HAVE_FLOAT64X;__HAVE_FLOAT64X_LONG_DOUBLE...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ=0;__NO_LONG_DOUBLE_MATH...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ=0;__STDC_VERSION__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC_PREREQ=0;__SUPPORT_SNAN__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUC__;__STDC__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GNUG__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GXX_EXPERIMENTAL_CXX0X__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __GXX_EXPERIMENTAL_CXX0X__;__STDC_VERSION__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __HAVE_FLOAT64X...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __HAVE_FLOAT64X;__HAVE_FLOAT64X_LONG_DOUBLE...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __HAVE_GENERIC_SELECTION...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __ILP32__;__x86_64__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __LDBL_COMPAT...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __LDBL_COMPAT;__MATH_DECLARING_FLOATN=0;__USE_ISOC99;__USE_ISOC99;__USE_XOPEN_EXTENDED...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __LDBL_COMPAT;__REDIRECT_NTH...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __LIBC...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __LONG_DOUBLE_MATH_OPTIONAL;__NO_LONG_DOUBLE_MATH...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __LP64__;__x86_64__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __MATH_DECLARING_FLOATN=0;__USE_MISC...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __MATH_DECLARING_FLOATN=0;__USE_MISC;__USE_XOPEN2K8;__USE_XOPEN_EXTENDED...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __MATH_DECLARING_FLOATN=0;__USE_MISC;__USE_XOPEN;__USE_XOPEN2K...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __NO_INLINE__;__OPTIMIZE_SIZE__;__OPTIMIZE__;__extern_inline...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __NO_LONG_DOUBLE_MATH...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __NO_LONG_DOUBLE_MATH;__SUPPORT_SNAN__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __NO_MATH_ERRNO__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __NO_MATH_INLINES;__OPTIMIZE__...',
  'Checking /home/chris/code/bottleneck/bottleneck/src/move_median/move_median.h: __NO_MATH_INLINES;__STRICT_ANSI__...',
  ... 60 more items
]
[
  `"/usr/include/stdio.h  407  information ConfigurationNotChecked: Skipping configuration '_GETOPT_H;__USE_GNU;__USE_POSIX2;__USE_POSIX_IMPLICITLY;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  410  information ConfigurationNotChecked: Skipping configuration '_GETOPT_H;__USE_GNU;__USE_POSIX2;__USE_POSIX_IMPLICITLY;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  451  information ConfigurationNotChecked: Skipping configuration '_GETOPT_H;__USE_GNU;__USE_POSIX2;__USE_POSIX_IMPLICITLY;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  456  information ConfigurationNotChecked: Skipping configuration '_GETOPT_H;__USE_GNU;__USE_POSIX2;__USE_POSIX_IMPLICITLY;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  '"/usr/include/bits/mathcalls-helper-functions.h  21  error syntaxError: syntax error"',
  '"/usr/include/bits/stdio-ldbl.h  30  error unknownMacro: There is an unknown macro here somewhere. Configuration is required. If __LDBL_REDIR1_DECL is a macro then please configure it."',
  '"/usr/include/bits/stdio-ldbl.h  23  error unknownMacro: There is an unknown macro here somewhere. Configuration is required. If __LDBL_REDIR_DECL is a macro then please configure it."',
  `"/usr/include/stdio.h  407  information ConfigurationNotChecked: Skipping configuration '__GLIBC_USE=0;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  410  information ConfigurationNotChecked: Skipping configuration '__GLIBC_USE=0;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  451  information ConfigurationNotChecked: Skipping configuration '__GLIBC_USE=0;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  456  information ConfigurationNotChecked: Skipping configuration '__GLIBC_USE=0;__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/string.h  410  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/math.h  322  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/math.h  325  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/math.h  327  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  21  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  25  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  30  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  33  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  36  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  39  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls-helper-functions.h  42  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  53  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  55  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  57  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  59  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  62  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  64  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  66  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  71  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  73  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  75  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  85  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  87  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  89  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  95  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  98  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  101  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  104  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  107  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  110  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  119  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  122  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  125  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  130  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  133  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  140  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  143  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  147  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  152  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  159  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  162  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  165  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  168  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  177  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  182  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  185  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  189  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  196  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  201  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  211  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  217  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  218  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  219  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  220  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  221  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  222  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  228  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  229  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  230  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  235  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  241  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  249  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  256  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  259  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  272  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  276  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  280  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  290  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  294  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  298  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  302  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  307  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  314  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  316  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  320  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  322  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  326  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  329  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  332  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  335  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/mathcalls.h  400  information ConfigurationNotChecked: Skipping configuration '__LDBL_COMPAT;__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  407  information ConfigurationNotChecked: Skipping configuration '__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  410  information ConfigurationNotChecked: Skipping configuration '__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  451  information ConfigurationNotChecked: Skipping configuration '__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/stdio.h  456  information ConfigurationNotChecked: Skipping configuration '__REDIRECT' since the value of '__REDIRECT' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/string.h  410  information ConfigurationNotChecked: Skipping configuration '__REDIRECT_NTH' since the value of '__REDIRECT_NTH' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly."`,
  `"/usr/include/bits/stdlib-bsearch.h  23  style variableScope: The scope of the variable '__idx' can be reduced. Warning: Be careful when fixing this message, especially when there are inner loops. Here is an example where cppcheck will write that the scope for 'i' can be reduced:`,
  'void f(int x)',
  '{',
  ... 40 more items
]
Line could not be parsed: void f(int x)
Line could not be parsed: {
Line could not be parsed:     int i = 0;
Line could not be parsed:     if (x) {
Line could not be parsed:         // it's safe to move 'int i = 0;' here
Line could not be parsed:         for (int n = 0; n < 10; ++n) {
Line could not be parsed:             // it is possible but not safe to move 'int i = 0;' here
Line could not be parsed:             do_something(&i);
Line could not be parsed:         }
Line could not be parsed:     }
Line could not be parsed: }
Line could not be parsed: When you see this message it is always safe to reduce the variable scope 1 level."
Line could not be parsed: void f(int x)
Line could not be parsed: {
Line could not be parsed:     int i = 0;
Line could not be parsed:     if (x) {
Line could not be parsed:         // it's safe to move 'int i = 0;' here
Line could not be parsed:         for (int n = 0; n < 10; ++n) {
Line could not be parsed:             // it is possible but not safe to move 'int i = 0;' here
Line could not be parsed:             do_something(&i);
Line could not be parsed:         }
Line could not be parsed:     }
Line could not be parsed: }
Line could not be parsed: When you see this message it is always safe to reduce the variable scope 1 level."
Line could not be parsed: void f(int x)
Line could not be parsed: {
Line could not be parsed:     int i = 0;
Line could not be parsed:     if (x) {
Line could not be parsed:         // it's safe to move 'int i = 0;' here
Line could not be parsed:         for (int n = 0; n < 10; ++n) {
Line could not be parsed:             // it is possible but not safe to move 'int i = 0;' here
Line could not be parsed:             do_something(&i);
Line could not be parsed:         }
Line could not be parsed:     }
Line could not be parsed: }
Line could not be parsed: When you see this message it is always safe to reduce the variable scope 1 level."
Completed lint scans...
```

**Comment by jbenden** [comment]
Hello,

I have found a work-around by setting the `c-cpp-flylint.cppcheck.verbose` option to `false`.

Best regards,
-Joe
